man/sysupdate.d.xml

```diff
@@ -76,6 +76,17 @@
     and have corresponding override directories for administrators (i.e.
     <filename>/etc/sysupdate.<replaceable>component</replaceable>.d/</filename>, etc).</para>
 
+    <para>System Administrators must take <emphasis>extreme</emphasis> care when overriding any transfer or
+    optional feature definitions, other than to turn on or off features!
+    As with any configuration defined in <filename>/usr</filename> and overridden in
+    <filename>/etc</filename>, an update to the host system can break the administrator overrides.
+    However, <command>systemd-sysupdate</command> is uniquely destructive: a broken configuration could
+    prevent the system from updating (best case), or completely destroy an installation by wiping the wrong
+    partition.
+    Distributions must take care to avoid breaking systems where overrides exist only to turn on or off
+    optional features; supporting (or choosing not to) everything else is up to distribution policy.
+    <emphasis>You have been warned.</emphasis></para>
```

Comment:
i am fine with warning, but I find this a bit *too* strong. i.e. the "extreme" and stuff is a bit too much.

What about this: let's use docbook's native constructs for this. i.e. `<caution><title>Warning!</title><para>…</para></caution>`

And then remove the `<emphasis>extreme</emphasize>` and the `You have been warned.

or in otherwords, let the typesetter solve this problem, not the text contents so much.

By toning this a bit down but at the same time offsetting this a bit from the rest of the text I think we get the message across quite well and appear a bit more professional I guess ;-)